boltons/formatutils.py

```diff
@@ -41,6 +41,7 @@
 import re
 from string import Formatter
 
+from .namedutils import namedtuple
```

Comment:
Done. It also occurs to me that I have also neglected to update the documentation.